app/bundles/EmailBundle/Entity/EmailRepository.php

```diff
@@ -48,12 +48,7 @@ public function getDoNotEmailList($leadIds = [])
 
         $results = $q->execute()->fetchAll();
 
-        $dnc = [];
-        foreach ($results as $r) {
-            $dnc[$r['id']] = strtolower($r['email']);
-        }
-
-        return $dnc;
+        return $results;
```

Comment:
It's public method. We cannot change response of result. The method could use another part of codebase or plugin. It's BC